packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -371,7 +370,7 @@ contract Deploy is Deployer {
         vm.broadcast(msg.sender);
         IProxy(payable(delayedWETHPermissionlessGameProxy)).upgradeToAndCall({
             _implementation: delayedWETHImpl,
-            _data: abi.encodeCall(IDelayedWETH.initialize, (msg.sender, ISuperchainConfig(superchainConfigProxy)))
+            _data: abi.encodeCall(IDelayedWETH.initialize, (msg.sender, ISystemConfig(systemConfigProxy)))
```

Comment:
Since this is only used once we can just do 
```suggestion
            _data: abi.encodeCall(IDelayedWETH.initialize, (msg.sender, ISystemConfig(artifacts.mustGetAddress("SystemConfigProxy")))
```